src/index.js

```diff
@@ -22,19 +22,23 @@ function onBootstrap(callback) {
   });
 }
 
-function ready(global) {
+function ready() {
   return new Promise((resolve, reject) => {
-    if (global.document.readyState === 'complete') {
-      resolve(global.document);
+    if (typeof global !== 'undefined' && global.process) { /* is NodeJS */
+      return resolve(true);
```

Comment:
Well, I guess you don't want code to continue. I would have just put the return on the next line. We can keep this as is.